### PR TITLE
Fit to finish priorities

### PR DESCRIPTION
**Pull request by swetha571**
# Type of change? 

- [ ] Bug fix
- [ ] New feature
- [x] Enhancement
- [ ] Breaking change (breaks backward compatibility)

# Description, Context, Motivation 
1. Titles change for maintenance, alarm grid  - https://github.com/Azure/pcs-remote-monitoring-webui/issues/480.
2. Alarm Log to Maintenance log  - https://github.com/Azure/pcs-remote-monitoring-webui/issues/816
3. Text Casing fix. 

...

**Checklist:**

- [ ] All tests passed
- [ ] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: src/common/lang.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft. All rights reserved.
 
 const lang = {
-  ACKNOWLEDGE: 'Acknowledge',
+  ACKNOWLEDGE: 'Ack',
```
Comment (spryor) [inline src/common/lang.js]: The name should match the string as much as possible so either change  `ACKNOWLEDGE` to `ACK` or add `ACK` as a key in the map.

File: src/common/lang.js [hunk 2]
```diff
@@ -178,7 +178,8 @@ const lang = {
   NO: 'No',
   NOOFDEVICES: 'NO. Devices',
   NOTE: 'Note: Any changes made to rules will only apply to future events on the affected devices.',
-  NOTIFICATIONTYPE: 'Notification Type',
+  NOTIFICATION : 'Notifications',
```
Comment (spryor) [inline src/common/lang.js]: missing 's' on `NOTIFICATION`

File: src/common/lang.js [hunk 3]
```diff
@@ -178,7 +178,8 @@ const lang = {
...
+  NOTIFICATIONTYPE: 'NOTIFICATION Type',
```
Comment (spryor) [inline src/common/lang.js]: Should NOTIFICATION be all caps? Seems like something that should be done in the CSS if so.

File: src/components/pages/maintenance/maintenance.js [hunk 4]
```diff
@@ -331,13 +332,20 @@ class MaintenancePage extends Component {
...
     const parentLink = <Link to={'/maintenance'}>{lang.MAINTENANCE}</Link>;
-    if (pathName.indexOf('/rule/') > 0) {
-      breadcrumbs = <span>{parentLink} &rsaquo; Alarm details</span>;
+    if (pathName.indexOf('/rule/') > 0 && (alarmsGridData || []).length > 0 ) {
+      alarmsGridData.forEach(rule => {
+      if (rule.id === this.props.params.id){
+        this.props.params.name = rule.name;
+      }
+    });
+      breadcrumbs = <span>{parentLink} <img src={ChevronRight}  alt={`${ChevronRight}`} className="chevron-right" /> {this.props.params.name}</span>;
```
Comment (spryor) [inline src/components/pages/maintenance/maintenance.js]: There are two spaces between `}  alt`

File: src/common/lang.js (hunk 3 above)
Comment (swetha571) [reply src/common/lang.js]: Nope. https://files.slack.com/files-pri/T0BD6KKA9-F836D6086/maintenance.png

File: src/components/pages/maintenance/maintenance.js [hunk 5]
```diff
@@ -331,13 +332,20 @@ class MaintenancePage extends Component {
...
+    if (pathName.indexOf('/rule/') > 0 && (alarmsGridData || []).length > 0 ) {
+      alarmsGridData.forEach(rule => {
+      if (rule.id === this.props.params.id){
+        this.props.params.name = rule.name;
+      }
+    });
+      breadcrumbs = <span>{parentLink} <img src={ChevronRight} alt={`${ChevronRight}`} className="chevron-right" /> {this.props.params.name}</span>;
     } else if (pathName.indexOf('/job/') > 0) {
-      breadcrumbs = <span>{parentLink} &rsaquo; System status details</span>;
+      breadcrumbs = <span>{parentLink} <img src={ChevronRight} alt={`${ChevronRight}`} className="chevron-right" />{this.props.params.jobId}</span>;
```
Comment (spryor) [inline src/components/pages/maintenance/maintenance.js]: Don't pass ChevronRight to the alt attribute. That will just render the src path on hover. Please pass an actual description.

**Pull request merged**